modules/friendlytag.js

```diff
@@ -168,14 +168,25 @@ Twinkle.tag.callback = function friendlytagCallback() {
 		case 'redirect':
 			Window.setTitle('Redirect tagging');
 
-			form.append({ type: 'header', label: 'Spelling, misspelling, tense and capitalization templates' });
-			form.append({ type: 'checkbox', name: 'redirectTags', list: Twinkle.tag.spellingList });
+			var i = 1;
+			$.each(Twinkle.tag.redirectList, function(title, content) {
+				form.append({ type: 'header', id: 'tagHeader' + i, label: title });
+				var subdiv = form.append({ type: 'div', id: 'tagSubdiv' + i++ });
+				$.each(content, function(subtitle, array) {
+					subdiv.append({ type: 'div', label: [ Morebits.htmlNode('b', subtitle) ] });
 
-			form.append({ type: 'header', label: 'Alternative name templates' });
-			form.append({ type: 'checkbox', name: 'redirectTags', list: Twinkle.tag.alternativeList });
+					var checkboxes = [];
+					$.each(array, function(k, item) {
+						checkboxes.push({ value: item.tag, label: '{{' + item.tag + '}}: ' + item.description });
+					});
```

Comment:
This can be more efficient using `map` as `checkboxes = array.map(item => ({ ... });`.

Reply:
Word!  Done.